lib/ansible/modules/system/timezone.py

```diff
@@ -55,13 +55,7 @@
   description: The differences about the given arguments.
   returned: success
   type: complex
-  contains:
-    before:
-      description: The values before change
-      type: dict
-    after:
-      description: The values after change
-      type: dict
+  sample: '{"before": {"name": "UTC"},<br>"after": {"name": "Asia/Tokyo"}}'
```

Comment:
This is intended (though is not mandatory) to give prettier result (see: https://github.com/ansible/ansible/issues/16665#issuecomment-293782315)

However, this doesn't pass the test, so I reverted it. Sorry.